### PR TITLE
feat: add replace special characters option to parse document

### PR DESCRIPTION
**Pull request by thomas9911**

### CODE REVIEW

File: functions/parse-document/1.1/function.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    },
+    {
+      "meta": {
+        "type": "Boolean",
+        "default": true
+      },
+      "advanced": true,
+      "name": "removeSpecialCharacters",
+      "label": "Remove Special Characters",
+      "info": "Remove special characters from the output."
```
Comment (robbplo) [inline functions/parse-document/1.1/function.json:52]: Could be clearer about what the option does. Decoding html-encoded spaces

Comment (thomas9911) [reply functions/parse-document/1.1/function.json]: updated

File: functions/parse-document/1.1/function.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    },
+    {
+      "meta": {
+        "type": "Boolean",
+        "default": true
+      },
+      "advanced": true,
+      "name": "decodeHtmlEntities",
+      "label": "Decode html entities",
+      "info": "Decode some html entities to normal characters such as: &nbsp; into ' '"
```
Comment (robbplo) [inline functions/parse-document/1.1/function.json:52]:
```suggestion
      "info": "Decode html entities to normal characters. Currently only converts &nbsp; into ' '"
```

Comment (thomas9911) [reply functions/parse-document/1.1/function.json]: updated

**Review by robbplo — APPROVED** [review]

**Pull request merged**